src/content/1.2/Types and Functions.tex

```diff
@@ -9,10 +9,10 @@ \section{Who Needs Types?}\label{who-needs-types}
 happily hitting random keys, producing programs, compiling, and running
 them.
 
-\begin{figure}
-\centering
-\fbox{\includegraphics[width=60mm]{images/img_1329.jpg}}
-\end{figure}
+\begin{wrapfigure}[10]{R}{0pt}
+\raisebox{0pt}[\dimexpr\height-0.75\baselineskip\relax]{
```

Comment:
When I copy-paste code I don't get I try to add links (say to https://tex.stackexchange.com/ questions).